packages/keychain/src/hooks/connection.ts

```diff
@@ -236,6 +238,18 @@ export function useConnectionValue() {
               message: "User logged out",
             });
           }
+
+          const connection = connectToParent<ParentMethods>({
```

Comment:
What's the purpose of this after logout?

Reply:
I see, we can just reuse `parent` instead of making another connection, updated. Feel free to merge if u see no other issues, thx!